src/pages/Contact.tsx

```diff
@@ -2,10 +2,55 @@ import * as React from "react";
 
 import Container from "../components/Container";
 import PageHeading from "../components/PageHeading";
+import styled from "../styled-components";
+import {
+  TwitterTimelineEmbed,
+  TwitterFollowButton
+} from "react-twitter-embed/dist/index.js";
+
+const TwitterBox = styled.div`
+  max-width: 400px;
+  max-height: 600px;
+  margin: 2em;
+`;
+
+const ContactInfoBox = styled.section`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 2em 0;
+
+  & > * {
+    flex-basis: 0;
+    flex-grow: 1;
+    margin: 0 2em;
+  }
+
+  & > *:last-child {
+    margin-top: 1em;
+  }
+`;
 
 const Contact = () => (
   <Container>
     <PageHeading>Contact</PageHeading>
+    <ContactInfoBox>
+      <div>
+        <h3>Darkside 2018 Contact</h3>
+        Jesse Kovacs
+        <br /> jjk007@live.unc.edu
```

Comment:
Here we should use the power of the internet and actually make this a link.
```suggestion
        <br />
        <a href="mailto:jjk007@live.unc.edu" rel="noopener noreferrer" target="_blank">jjk007@live.unc.edu</a>
```